docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -66,55 +71,85 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
-1. Click **Manifest**, then define the required Application Role values for Grafana: Viewer, Editor, or Admin. If not defined, all users will have the Viewer role. Every role requires a unique ID which you can generate on Linux with `uuidgen`, and on Windows through Microsoft PowerShell with `New-Guid`.
-
-1. Include the unique ID in the configuration file:
-
-   ```json
-   	"appRoles": [
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana org admin Users",
-   				"displayName": "Grafana Org Admin",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Admin"
-   			},
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana read only Users",
-   				"displayName": "Grafana Viewer",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Viewer"
-   			},
-   			{
-   				"allowedMemberTypes": [
-   					"User"
-   				],
-   				"description": "Grafana Editor Users",
-   				"displayName": "Grafana Editor",
-   				"id": "SOME_UNIQUE_ID",
-   				"isEnabled": true,
-   				"lang": null,
-   				"origin": "Application",
-   				"value": "Editor"
-   			}
-   		],
-   ```
-
-1. Go to **Azure Active Directory** and then to **Enterprise Applications**. Search for your application and click on it.
-
-1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
-
+2. You can define the required Application Roles for Grafana [using the Azure Portal](#configure-application-roles-for-grafana-on-the-azure-portal) or [using the manifest file](#configure-application-roles-for-grafana-in-the-manifest-file).
+
+3. Go to **Azure Active Directory** and then to **Enterprise Applications**. Search for your application and click on it.
+
+4. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
+
+#### Configure Application roles for Grafana on the Azure Portal
+
+In this seciont we'll go through how to setup basic Appplication roles for Grafana from the Azure Portal. More info can be found [here](https://learn.microsoft.com/en-us/entra/identity-platform/howto-add-app-roles-in-apps):
+
+1. Go to **App Registrations**. Search for your application and click on it.
+
+1. Click on **App roles** and then **Create app role**.
+
+1. Define a role corresponding to each Grafana role: Viewer, Editor, and Admin. 
+
+    1. Choose a **Display name** for the role. For example, "Grafana Editor".
+            
+    1. Set the **Allowed member types** to **Users/Groups**.
+
+    1. Ensure that the **Value** field matches the Grafana role name. For example, "Editor".
+
+    1. Choose a **Description** for the role. For example, "Grafana Editor Users".
+
+    1. Click **Apply**.
+
+#### Configure Application roles for Grafana in the manifest file
+
```

Comment:
Add an introduction here that explains when and why I would use this approach over the Azure portal.

Reply:
It's based on the user's preference, I added this: "If you prefer to configure the application roles for Grafana in the manifest file, complete the following steps:"